### ISSUE TITLE
Create service form fieldset doesn't have legend

### ISSUE THREAD

**Issue by msecret** [18F/cg-deck#371]
>
Fieldset does not contain a legend element. All fieldsets should contain a legend element that describes a description of the field group.

Related #37

**Comment by msecret** [comment]
Closing to one issue.
